.eslintrc.js

```diff
@@ -110,19 +104,14 @@ module.exports = {
       }
     },
     {
-      files: [
-        'test/aria-practices/**/*.js'
-      ],
+      files: ['test/aria-practices/**/*.js'],
       env: {
         mocha: true
       }
     },
     {
       files: ['test/**/*.js'],
       excludedFiles: 'test/aria-practices/**/*.js',
-      parserOptions: {
```

Comment:
We can now use es6 syntax in tests

Reply:
True, but unrelated to this PR. Please pull this out.